### DISCUSSION TITLE
Lumped model heat structure maximum temperature too high

### DISCUSSION THREAD

**Discussion by asm-fakhrul** [idaholab/moose#25263] [Q&A Modules: General]
After dividing my 1 MW lumped core into three rings, the maximum temperature in each heat structure is too high (between 2300 and 3000 K). Apparently, others have seen similar results in the past. Excerpting from a paper (https://publications.anl.gov/anlpubs/2019/10/156426.pdf) that reports their SAM lumped model results for the 600 MW MHTGR design: 

"It was quickly found that the simplified ring model did not produce correct solid temperatures, which is too high as the thickness of heat structures are too large. A detailed ring model is then developed."

The authors then updated their lumped model from having 11 coolant channels and 22 heat structures to having 33 coolant channels and 66 heat structures.

Is a three-ring lumped model in MOOSE not sufficient to capture the thermal hydraulics reliably? Are there any recommendations as to how many rings should/must be used for a good lumped model assumption?

Thank you!

**Comment by GiudGiud** [comment]
Hello

I think the best people to comment on this are the authors of this report
@rhu @lingzou 

Guillaume

**Reply by asm-fakhrul** [reply]
Gently trying to bring the thread to the authors' attention.

**Reply by lingzou** [reply]
Hi @asm-fakhrul, to answer your question, I may need your help to clarify some context of your question.

The ring-model developed in the report you have cited was for prismatic HTGR. Is your reactor design also prismatic HTGR? Are you using MOOSE or SAM?
It's difficult to answer your question on number of rings are sufficient without knowing pretty nothing about your problem on hand.

**Reply by alfoa** [reply]
@lingzou see below:

**Comment by asm-fakhrul** [comment]
1. Yes, it's a prismatic HTGR design.
2. I am using MOOSE.

**Reply by lingzou** [reply]
Thank you @asm-fakhrul. In that case, I am afraid that I may not be able to provide much specific comments, since SAM (although a MOOSE-based application) has its specially built-in components to enable system analysis of reactor systems. When speaking of 'rings,' we may talk about completely different things in SAM compared what you mean in your MOOSE simulations.

**Reply by asm-fakhrul** [reply]
Okay. Thank you for your response. I look forward to insight from others.

**Reply by GiudGiud** [reply]
@joshuahansel

**Reply by joshuahansel** [reply]
I'm a bit unclear on the model here. Can you clarify your model geometry/setup here? You're coupling 2D heat structures to flow channels? In this "ring model", how are all of the flow channels modeled? I don't know how this works.

In any case, this sounds like a question of experience making simplified 2D RZ models of reactor cores. I have never tried this, so I doubt I will be of much help. I can answer questions about how the components work and what is possible.

**Reply by GiudGiud** [reply]
@snschune has experience in this I think
